spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -497,6 +487,96 @@ object Recurrent extends ContainerSerializable {
     new Recurrent[T](batchNormParams)
   }
 
+  /**
+   * set the cells' output and gradInput to recurrent's output and gradInput
+   * to decrease the copy expense.
+   * Copy src tensor to dst tensor along timeDime, default timeDime 2, batchDim 1
+   * @param src
+   * @param dst
+   */
+  private[bigdl] def copy[@specialized(Float, Double) T: ClassTag](
+    src: ArrayBuffer[Tensor[T]], dst: Tensor[T], offset: Int): Unit = {
```

Comment:
why do we need the offset parameter?

Reply:
I just keep the original interface, from the whole code, maybe it can be deleted.